src/core/settings_core.json

```diff
@@ -1881,6 +1881,23 @@
       "VariableName": "useFp16GenMips",
       "Description": "If mipGenUseFastPath == true and this is true - use the fp16 single-pass GenMips compute pass."
     },
+    {
+      "Name": "maxMappedPoolsSize",
+      "Tags": [
+        "Resource Settings",
+        "Performance"
+      ],
+      "Defaults": {
+        "Default": 0
```

Comment:
Thanks for leaving the default at zero here!  This minimizes the risk of regressions being introduced and gives the opportunity for later tuning.

Reply:
Yes, during my earlier testing I discovered that some the CTS tests were failing, but I have already update the change to fix that.